src/common/hobject.h

```diff
@@ -327,6 +329,22 @@ template<> struct hash<hobject_t> {
 
 std::ostream& operator<<(std::ostream& out, const hobject_t& o);
 
+template <> struct fmt::formatter<hobject_t> {
+  constexpr auto parse(format_parse_context& ctx)
+  {
+    return ctx.begin();
+  }
+
+  template <typename FormatContext> auto format(hobject_t const& ho, FormatContext& ctx)
+  {
+    // for now - just use the existing operator
+    stringstream sst;
+    sst << ho;
```

Comment:
this actually hurts the performance.

Reply:
@tchaikov:
Agree on the performance, and will fix to include all the details
(these commits are just for you to OK the general idea and the change to the lib version).

Re using fmt/ostream directly: tried that, and immediately got into an endless loop during execution
(the ostream implementation tried to use fmt...). I couldn't find a clear way around (which may be the reason some
users advocate against using this option).